sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/DiagnosticsProvider.java

```diff
@@ -270,8 +269,9 @@ private <T> void endSpanCore(
 
         switch (signal.getType()) {
             case ON_COMPLETE:
+                end(cosmosCtx, statusCode, 0, actualItemCount, requestCharge, diagnostics,null, context, true);
```

Comment:
The CI is complaining of a fall-through case. Missing `break`.

Reply:
Fixed